config/v3_0/types/mode.go

```diff
@@ -24,3 +26,13 @@ func validateMode(m *int) error {
 	}
 	return nil
 }
+
+func validatePermissionBits(m *int) error {
```

Comment:
Could this just be part of `validateMode`?

Reply:
That's my plan earlier, however, we need to throw a warning in this case and `validateMode` throws an error (which might halt the system's execution in this case)

Reply:
Ah, yeah, you're right.  Let's rename the function then to something more accurate, maybe `validateModeSpecialBits`.